hydra_base/lib/network.py

```diff
@@ -2810,9 +2818,13 @@ def clone_network(network_id,
     newnet.projection = ex_net.projection
     newnet.created_by = user_id
 
-    newnet.set_owner(user_id)
-    if recipient_user_id is not None:
-        newnet.set_owner(recipient_user_id)
+    #if true, the the creator will see this network in their project.networks.
+    if creator_is_owner is True and user_id != recipient_user_id:
+        newnet.set_owner(user_id)
+
+    #set the owner to the recipient. THis can be either the requesting user id (user_id)
+    #or an explicitly defined user. 
+    newnet.set_owner(recipient_user_id)
```

Comment:
This isn't the case, because the statement above only occurs when user_id != recipient_user_id. 
The recipient_user_id is set to user_id earlier where recipient_user_id is None. 
Therefore if both of these set_owner functions arecalled, they must be done with different user_ids.

Reply:
This is resolved, the error in my above comment is that ownership is not exclusive: `.set_owner()` does not set *the* owner but adds its argument to the set of owners.  In case 1 above, both creator and recipient are made owners, which is the intent specified by the `creator_is_owner` flag.